db/migrate/20180703175739_modified_first_and_last_not_null.rb

```diff
@@ -0,0 +1,6 @@
+class ModifiedFirstAndLastNotNull < ActiveRecord::Migration[5.2]
+  def change
+  change_column :users, :first_name, :string, null: false
```

Comment:
Just check the indentation here, a really minor thing.
We tend to use 2 spaces for indetation.